<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="SimpleFlow" name="label">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>240</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="verticalGroupBox_2">
              <property name="title">
               <string>Choix du niveau</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QRadioButton" name="radioButton_Level1">
                 <property name="text">
                  <string>Niveau 1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_Level2">
                 <property name="text">
                  <string>Niveau 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_Level3">
                 <property name="text">
                  <string>Niveau 3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_restart">
                 <property name="text">
                  <string>Jouer !</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="verticalGroupBox_2">
              <property name="title">
               <string>Cette partie</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_spheresRestantes">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sph√®res restantes : 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_tempsTotal">
                 <property name="text">
                  <string>Temps total :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_currentsphere">
                 <property name="text">
                  <string>Temps manche :</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="verticalGroupBox">
            <property name="title">
             <string>Records</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="labelHeader">
               <property name="font">
                <font>
                 <family>Courier</family>
                </font>
               </property>
               <property name="text">
                <string>Temps|Niv.|Pseudo </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ListRecords" name="listView">
               <property name="font">
                <font>
                 <family>Courier</family>
                </font>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MyGLWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimpleFlow</class>
   <extends>QLabel</extends>
   <header>simpleflow.h</header>
  </customwidget>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ListRecords</class>
   <extends>QListView</extends>
   <header>listrecords.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
